cedar-policy-cli/Cargo.toml

```diff
@@ -28,6 +28,7 @@ partial-validate = ["cedar-policy/partial-validate"]
 assert_cmd = "2.0"
 tempfile = "3"
 glob = "0.3.1"
+predicates = "3.1.0"
```

Comment:
assert_cmd allows predicates to match on stdout. assert_cmd already depends on predicates, so I don't think this affects the dependency tree much.

Reply:
Look fine to me.